python2.7/run/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN rm /var/runtime/awslambda/runtime.so
 ADD runtime-mock.py /var/runtime/awslambda/runtime.py
 
 # Not sure why permissions don't work just by modifying the owner
-RUN rm -rf /tmp && mkdir /tmp && chown -R sbx_user1051:495 /tmp && chmod 700 /tmp
+RUN rm -rf /tmp && mkdir /tmp && chown -R sbx_user1051:495 /tmp && chmod 700 /tmp && chmod -R 0777 /var/runtime
```

Comment:
In the real lambda environment `/var/runtime/awslambda/runtime` is a compiled binary. In our case it's a .py file. We need the user we are running under to be able to create the `/var/runtime/awslambda/runtime.pyc` file.

Now I realise it's really hackish to give such wide permissions, and would be glad to implement a better solution. Ideas welcome :D